static/js/unread_ops.js

```diff
@@ -127,8 +142,9 @@ function process_newly_read_message(message, options) {
 
 export function mark_as_unread_from_here(
     message_id,
-    include_message = true,
+    include_anchor = true,
     messages_marked_unread_till_now = 0,
+    num_after = INITIAL_BATCH_SIZE - 1,
```

Comment:
This inserted a 4th positional parameter to `mark_as_unread_from_here`, but failed to update the call below, which means `num_after` becomes `narrow` and `narrow` becomes `undefined`.

https://github.com/zulip/zulip/blob/f0d87fcf6e9e04967e29ebabe2b2d75e6f9a4377/static/js/unread_ops.js#L218-L229